wallets/index.js

```diff
@@ -196,17 +196,16 @@ export function WalletsProvider ({ children }) {
 
   // provides priority sorted wallets to children, a function to reload local wallets,
   // and a function to set priorities
+  const value = useMemo(() => ({
+    wallets,
+    reloadLocalWallets,
+    setPriorities,
+    onVaultKeySet: syncLocalWallets,
+    beforeDisconnectVault: unsyncLocalWallets,
+    removeLocalWallets
+  }), [wallets, reloadLocalWallets, setPriorities, syncLocalWallets, unsyncLocalWallets, removeLocalWallets])
```

Comment:
Did this `useMemo` make a difference in the render count?

Reply:
I think so:

https://github.com/user-attachments/assets/d03e7dd6-d289-46cb-82da-dff978532760